app/services/submission_service.rb

```diff
@@ -185,6 +187,19 @@ def submit!(submission)
         .notify_user(submission.id)
     end
 
+    def resubmit!(submission)
+      if submission.assigned_to
+        assigned_admin = AdminUser.find_by(gravity_user_id: submission.assigned_to)
+
+        if assigned_admin && Convection.unleash.enabled?(
+          "onyx-admin-submission-approved-email",
+          Unleash::Context.new(user_id: assigned_admin.gravity_user_id.to_s)
```

Comment:
I haven't created a new feature flag for the new email yet and wonder whether we need one. In my opinion, a feature flag shouldn't be necessary because this email is only sent to admins, and we can easily roll back or fix the code.

@dblandin, you have created a feature flag for the "submission approved" admin email in https://github.com/artsy/convection/pull/1512. Do you think it would be a good idea to create a feature flag for the "Resubmitted" email as well?

Reply:
yeah, I think we can easily rollback if anything. Do you know if admins know about upcoming change?

Reply:
An Unleash feature flag that can be enabled per user ID might be good to test the new email before sending it to admins. So I think I will create one (and remove it when the feature is released).

Thanks for the reminder to inform about the new email before releasing it 🙇 I haven't done it yet.